cfme/fixtures/pytest_store.py

```diff
@@ -148,9 +148,9 @@ def write_line(self, line, **kwargs):
 store = Store()
 
 
-def pytest_namespace():
-    # Expose the pytest store as pytest.store
-    return {'store': store}
+# def pytest_configure(config):
+#     # Expose the pytest store as pytest.store
+#     pytest.store = store
```

Comment:
Seeking advice from @RonnyPfannschmidt on this replacement, commented out and direct imports of the singleton used for now.

Reply:
Any resolution on this?

Reply:
The singleton in the fixture will work just fine